### PR TITLE
#37 add CLI support for inserting TOCs inline into files

### PR DESCRIPTION
**Pull request by RichardBradley**

### CODE REVIEW

**Comment by RichardBradley** [comment]
Fix for #37

File: cli.js [hunk 1]
```diff
@@ -3,19 +3,43 @@
...
+  process.exit(1);
+}
+
+if (args.i && args.json) {
+  console.error('markdown-toc: you cannot use both --json and -i');
+  process.exit(1);
+}
+
+if (args.i && args._[0] === "-") {
+  console.error('markdown-toc: you cannot use both -i with stdin input');
```
Comment (jonschlinkert) [inline cli.js]: I think `both -i` is a grammatical error. should it be `both -i and -`?

Comment (RichardBradley) [inline cli.js]: agreed, fixed

**Comment by RichardBradley** [comment]
Does this look OK to you now?

If you merge this in December, I get brownie points on http://24pullrequests.com/ ;-)

**Comment by jonschlinkert** [comment]
awesome, lol sorry over the past two weeks I had the flu and then another cold back-to-back. Still programming but not as much as usual. (that's what happens when you have two kids in daycare lol). 

I'll definitely get this merged in the next day or so. thanks for the reminder!

**Pull request merged**